src/App.tsx

```diff
@@ -15,47 +15,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
 export const App: React.FC = () => {
+  const [onReverse, handleReverse] = useState(false);
+  const [onSortType, handleSortByType] = useState(SortType.NONE);
```

Comment:
Use `handle...` for handlers `onClick` `OnChange` `OnMouseDown` etc. 
Use `setTheSameNameOfVariable` for setters in useState. Ex. `[value, setValue]`, `[count, setCount]`, `[isLoading, setIsLoading]` etc.

```suggestion
  const [sortType, setSortByType] = useState(SortType.NONE);
```